assets/js/app.js

```diff
@@ -156,17 +156,17 @@ if ( (userCity == "^\s\.\-[\s*\.\-A-Z\s*\.\-a-z\s*\.\-_ ]\.\- \s{1,50}") ){
 
   (userCity).innerHTML = "Not a valid response";
   status = false;
-  badInput = badInput + "City<br>"  //  *********** added by fdh to create parameter for function call  ************
+  badInput += "City<br>"  //  *********** added by fdh to create parameter for function call  ************
 } else{
   status = true;
 }
 
 //  *********** showModal moved here by fdh to encomass all input data ************
-if (badInput != "") {  
+if (status="false") {  
```

Comment:
You want `status == false`, because `status = "false"` sets the `status variable equal to a string that holds the value of the word "false", instead of checking if `status` is equal to the boolean value of `false`